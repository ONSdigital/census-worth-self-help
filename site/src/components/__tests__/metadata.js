import React from "react"
import renderer from "react-test-renderer"

import Metadata from "../metadata"

describe("Metadata", () => {
  it("renders correctly with title", () => {
    const tree = renderer
      .create(<Metadata>a title</Metadata>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly without a title", () => {
    const tree = renderer
      .create(<Metadata></Metadata>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})