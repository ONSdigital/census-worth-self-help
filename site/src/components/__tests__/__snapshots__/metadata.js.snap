// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Metadata renders correctly with title 1`] = `null`;

exports[`Metadata renders correctly without a title 1`] = `null`;
